src/components/interface/Nav/Nav.tsx

```diff
@@ -42,11 +42,16 @@ const Nav: React.FunctionComponent = React.memo(() => (
     </NavLink>
 
     <NavLink
+      isNew={true}
       subLinks={[
         {
           text: 'LP POOLS',
           link: `/${routes.POOLS}`,
         },
+        {
+          text: 'LP OPTIMIZERS',
```

Comment:
Minor detail but the copy everywhere else spells Optimiser with an s. Good to be consistent with this.

Reply:
Thank you